dspace/modules/xmlui/src/main/java/ar/edu/unlp/sedici/xmlui/xsl/XslExtensions.java

```diff
@@ -165,5 +168,12 @@ public static String encodeURL(String urlToEncode) throws UnsupportedEncodingExc
 	public static String getFileExtension(String filename) {
 		return filename.substring(filename.lastIndexOf(".") + 1).toUpperCase();
 	}
+
+	//Total de items disponibles en el repo, redondeado hacia abajo a la unidad de centena inferior mas cercana.
+	//Es decir, si hay 67160 items redondea a 67100.
+	public static int availableItemsCount() throws SQLException {
+		int totalItems = Utils.countAvailableItems(new Context());
```

Comment:
@santit96 te parece si para evitar el calculo aca, lo hacemos en la consulta a la base de datos.

`SELECT count(*) / 100 * 100 as total FROM item WHERE in_archive='1' and withdrawn='0'`

Reply:
hecho